signac/common/config.py

```diff
@@ -99,45 +98,33 @@ def read_config_file(filename, configspec=None, *args, **kwargs):
     return config
 
 
-def load_config(root=None, local=False):
-    """Load configuration, searching upward from a root path if desired.
+def load_config(root=None):
+    """Load configuration at root directory.
 
     Parameters
     ----------
     root : str
-        The path from which to begin searching for config files.
-    local : bool, optional
-        If ``True``, only search in the provided directory and do not traverse
-        upwards through the filesystem (Default value: False).
+        The path from which to load the local config.
 
     Returns
     --------
     :class:`Config`
         The composite configuration including both local and global config data
-        if requested.
+        if requested. Note that because this config is a composite,
+        modifications to the returned value will not be reflected in the files.
     """
     if root is None:
         root = os.getcwd()
     config = Config(configspec=cfg.split("\n"))
-    if local:
-        search_func = _search_local
-    else:
-        for fn in _search_standard_dirs():
+
+    # Add in any global or user config files. For now this search only finds user-specific
```

Comment:
Slightly redundant...?
```suggestion
    # Add in any global or user config files. For now this only finds user-specific
```